src/Linux/Avalonia.LinuxFramebuffer/Input/LibInput/LibInputBackend.cs

```diff
@@ -9,36 +9,38 @@ namespace Avalonia.LinuxFramebuffer.Input.LibInput
 {
     public partial class LibInputBackend : IInputBackend
     {
-        private IScreenInfoProvider _screen;
-        private IInputRoot _inputRoot;
-        private TouchDevice _touch = new TouchDevice();
+        private IScreenInfoProvider? _screen;
+        private IInputRoot? _inputRoot;
         private const string LibInput = nameof(Avalonia.LinuxFramebuffer) + "/" + nameof(Avalonia.LinuxFramebuffer.Input) + "/" + nameof(LibInput);
-        private Action<RawInputEventArgs> _onInput;
-        private Dictionary<int, Point> _pointers = new Dictionary<int, Point>();
+        private Action<RawInputEventArgs>? _onInput;
+        private readonly LibInputBackendOptions? _options;
 
         public LibInputBackend()
         {
-            var ctx = libinput_path_create_context();
-            new Thread(() => InputThread(ctx))
-            {
-                IsBackground = true
-            }.Start();
+            _options = default;
+        }
+
+        public LibInputBackend(LibInputBackendOptions options)
+        {
+            _options = options;
         }
 
-        private unsafe void InputThread(IntPtr ctx)
+        private unsafe void InputThread(object? state)
```

Comment:
Can we pass context IntPtr and options class separately here? Since LibInputBackendOptions are supposed to be a user-supplied options dto class, not a holder for native resources. Just use a closure in Thread ctor.